tck/tck-common/src/main/java/jakarta/jsonp/tck/lib/harness/Fault.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.jsonp.tck.lib.harness;
+
+import junit.framework.AssertionFailedError;
+
+public class Fault extends AssertionFailedError {
```

Comment:
This exception class isn't really needed. Can just use the standard junit exceptions

Reply:
I did it in that way to dramatically decrease the modifications in the CTS tests. There are many methods throwing that Fault.

We need to live for some time with this copy-pasted tests, so if a modification happens in jakartaee-tck, we need to update this tests too. For the human eye it will be easier to see differences.